src/components/UserPortal/ContactCard/ContactCard.tsx

```
@@ -1,94 +1,102 @@
+        defaultValue: 'contact-card-{{id}}',
+        id,
+      })}
+      ariaLabel={t('contact.card_aria', 'Contact card')}
+      imageSlot={imageSlot}
+      actionsSlot={actionsSlot}
+      className={styles.contactCardWrapper}
+    >
       <div
-        className={`${styles.contact} ${
-          isSelected ? styles.bgGreen : styles.bgWhite
+        role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)